.eggs/README.txt

```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
```

Comment:
I think an issue occurred with the git commit process,
It seems like your entire directory was pushed including undesired files